pkg/controller/projectclaim/projectclaim_controller.go

```diff
@@ -83,9 +80,25 @@ func (r *ReconcileProjectClaim) Reconcile(request reconcile.Request) (reconcile.
 	}
 
 	adapter := NewCustomResourceAdapter(instance, reqLogger, r.client)
+	result, err := r.ReconcileHandler(adapter)
+	if err != nil {
+		message := err.Error()
+		reason := "ReconcileFailed"
+		// Update the ProjectClaimConditionCRD
+		return result, adapter.SetProjectClaimCondition(corev1.ConditionTrue, reason, message)
```

Comment:
should we return the actual err instead of StatusUpdate() err?

Reply:
Definitely! I think errors occuring here, we should just ignore.

Reply:
anyway, just ignored